commit dafcc815b8f604b95e57986d3c3288b02f8d20d2
Author: Peter Penz <peter.penz19@gmail.com>
Date:   Sat Apr 7 23:11:16 2012 +0200

    Fix drag & drop cursor issue
    
    When dragging a file within the same Dolphin window and the Ctrl, Shift
    or Alt-key is pressed the shape of the cursor was not changed to
    indicate the copy/move/link-operation.
    
    BUG: 293850
    FIXED-IN: 4.8.3

diff --git a/dolphin/src/kitemviews/kitemlistcontroller.cpp b/dolphin/src/kitemviews/kitemlistcontroller.cpp
index 1ad760a..79a6ecf 100644
--- a/dolphin/src/kitemviews/kitemlistcontroller.cpp
+++ b/dolphin/src/kitemviews/kitemlistcontroller.cpp
@@ -749,6 +749,8 @@ bool KItemListController::dragMoveEvent(QGraphicsSceneDragDropEvent* event, cons
         return false;
     }
 
+    event->acceptProposedAction();
+
     KItemListWidget* oldHoveredWidget = hoveredWidget();
 
     const QPointF pos = transform.map(event->pos());
